### DISCUSSION TITLE
remote

### DISCUSSION THREAD

**Discussion by msw-home** [pybricks/support#1395] [Q&A]
Moin, kann man irgendwie 2 Fernbedienungen (remote) gleichzeitig an einem Hub konfigurieren und benutzen?

**Comment by laurensvalk** [comment]
You can use just one remote, but you could add some logic to add an extra 'mode'.

For example, you could use the +/- for driving at first. But once your program detects that the green button was pressed, you could make it do something else for the +/- buttons, like operating a gripper.

**Reply by msw-home** [reply]
Gibt es dazu ein Beispiel im Blockprogramm geschrieben?

**Reply by BertLindeman** [reply]
Just as a very basic example.

It uses the left and right (Red) remote buttons to switch between "drive mode" and "work mode".
The program does "nothing" but show the mode and in drive mode shows the button pressed.

I have just put this together as example without further usage. 
(It is my first pybricks block program ever....)

![afbeelding](https://github.com/pybricks/support/assets/8142081/599edf79-a15a-4e59-b526-f885cd64cea0)

Have fun,

Bert

**Reply by msw-home** [reply]
Moin, schon mal vielen Dank vorab. Das muss ich erstmal verarbeiten und verstehen! Zur Erklärung, ich bin 68 Jahre und versuche mich an der Blockprogrammierung um meinen Enkel 7 Jahre zu unterstützen. Er baut Modelle und ich darf programmieren! LG Manfred

**Comment by BertLindeman** [comment]
> Moin, schon mal vielen Dank vorab. Das muss ich erstmal verarbeiten und verstehen! Zur Erklärung, ich bin 68 Jahre und versuche mich an der Blockprogrammierung um meinen Enkel 7 Jahre zu unterstützen. Er baut Modelle und ich darf programmieren! LG Manfred

Maybe swap your functions 😉 You build and delegate the programming. These kids are SO fast learning.
Just yoking, you know.

Perhaps a step back? First program a model and operate it with a remote with only the drive and steer functions?
Just to learn how to do the programming?
This is a more simple example. 
The steering angle **should** be determined in the program, but I want to keep it as simple as I can.
So determine the steering angle by testing and set the variable in the program.

![pybricks_program_2024-01-25T16_18_42](https://github.com/pybricks/support/assets/8142081/2522cd0a-a11e-4938-ae64-5ce9346ffe99)
